src/cli/ionic-config.ts

```diff
@@ -1,17 +1,19 @@
 import { getCompilerSystem } from './state/stencil-cli-config';
 import { readJson, uuidv4 } from './telemetry/helpers';
 
-export const DEFAULT_CONFIG = (file: boolean = false) => {
-  return getCompilerSystem().resolvePath(`${getCompilerSystem().homeDir()}/.ionic${file ? '/config.json' : ''}`);
-};
+export const default_config = () =>
```

Comment:
Should these be `defaultConfig` and `defaultConfigDirectory`?

Reply:
Patched!